.github/workflows/pull-request.yml

```diff
@@ -36,12 +36,18 @@ jobs:
       - name: Check out forklift repository
         uses: actions/checkout@v3
 
+      - name: Checkout forkliftci
+        uses: actions/checkout@v3
+        with:
+          repository: kubev2v/forkliftci
+          ref: v2.0
+
       - name: Build and setup everything with bazel
         id: forkliftci
-        uses: kubev2v/forkliftci@v2.0
+        uses: kubev2v/forkliftci/ci/build-and-setup@main
```

Comment:
don't we need to take it also from v2.0 rather than `main`?
same question for the actions access below

Reply:
bumped to v3.0 which the updated actions.